setup.bat

```diff
@@ -0,0 +1 @@
+mvn package
```

Comment:
Nope, I'm sure we have developers that use it as well - That said, you can use cygwin on Windows.